cachehandler.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"sync"
+)
+
+// Singleton
+var bc bookCache
+var cacheOnce sync.Once
+
+// Singleton method
```

Comment:
Why is this a singleton?
The operation of creating a new client is not expensive